UnitsNet/Scripts/UnitDefinitions/Flow.json

```diff
@@ -64,10 +64,12 @@
       "PluralName": "LitersPerMinute",
       "FromUnitToBaseFunc": "x/60000.00000",
       "FromBaseToUnitFunc": "x*60000.00000",
+	  "Prefixes": [ "Nano", "Micro", "Milli", "Centi", "Deci", "Kilo"],
       "Localization": [
         {
           "Culture": "en-US",
-          "Abbreviations": [ "LPM" ]
+          "Abbreviations": [ "LPM" ],
+		  "AbbreviationsWithPrefixes": ["nLPM", "μLPM", "mLPM", "cLPM", "DLPM", "KLPM"]
```

Comment:
It should be "dLPM" shouldn't it? It's probably not even a widely used abbreviation, but we should at least get the prefixes right.